app/graphql/types/rating_system.rb

```diff
@@ -0,0 +1,5 @@
+class Types::RatingSystem < Types::BaseEnum
+  value 'SIMPLE', 'Smileys', value: 'simple'
+  value 'REGULAR', '1-10 whole numbers', value: 'regular'
+  value 'ADVANCED', '1-100 whole numbers', value: 'advanced'
```

Comment:
I'm 99% sure this is wrong and it's actually 1-20

Reply:
I'll double check it.